razorpay-payments.php

```diff
@@ -52,6 +52,20 @@ public function __construct()
             $this->msg['message'] = '';
             $this->msg['class'] = '';
 
+            // Supports woocommerce subscriptions
+            $this->supports = array(
+                'products',
+                'subscriptions',
+                'subscription_cancellation',
+                'subscription_suspension',
+                'subscription_reactivation',
+                'subscription_amount_changes',
+                'subscription_date_changes',
+                'subscription_payment_method_change'
+            );
+
+            add_action('woocommerce_scheduled_subscription_payment', array($this, 'process_subscription'));
```

Comment:
@captn3m0 Using it for now. Will remove it once we get the webhook code working.